src/main/archetype/ui.apps/pom.xml

```diff
@@ -56,28 +56,20 @@
                 <artifactId>filevault-package-maven-plugin</artifactId>
                 <extensions>true</extensions>
                 <configuration>
-                    <group>${packageGroup}</group>
+                    <group>${groupId}</group>
+                    <name>${appsFolderName}.ui.apps</name>
+                    <packageType>application</packageType>
+                    <accessControlHandling>merge</accessControlHandling>
+                    <properties>
+                        <cloudManagerTarget>none</cloudManagerTarget>>
+                    </properties>
                     <embeddeds>
```

Comment:
I believe this is missing a reference to the projects Repository Structure Package?

Reply:
Added in https://github.com/adobe/aem-project-archetype/pull/256/commits/b4b94d13dfd5506e5e2d5d617b41fa3de66fb10a#diff-9a5eca8b7f42611af149eeabb4c3686eR62